sst/config/config.py

```diff
@@ -0,0 +1,68 @@
+import sst
```

Comment:
Again, this file's name should be more descriptive of what it is modelling or if it is an example config

Reply:
Done.